src/test/java/uk/gov/hmcts/reform/prl/services/cafcass/CaseDataServiceTest.java

```diff
@@ -72,6 +73,7 @@ public void setUp() {
         when(authTokenGenerator.generate()).thenReturn(s2sToken);
     }
 
+    @Ignore
```

Comment:
This is throwing code smell - https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=1645&id=prl-cos-api&open=AYoc8tfAnseCIXeqSRX5